.githooks/pre-commit

```diff
@@ -1,8 +1,39 @@
 #!/bin/bash
 cd "$(git rev-parse --show-toplevel)"
+# Pylint Path is Micky's local path, feel free to change in PR 
+PYLINT = "c:\users\micky\appdata\local\programs\python\python310\lib\site-packages" 
 ESLINT="client/node_modules/.bin/eslint"
 pwd
 
+if [[ ! -x "$PYLINT" ]]; then
+	  printf "\t\033[41mPlease install Pylint\033[0m (pip install pylint)\n"
+	    exit 1
+    fi
+
+    STAGED_FILES=($(git diff --cached --name-only --diff-filter=ACM | grep ".py\{0,1\}$"))
```

Comment:
I'm not a regex expert so I don't know what the \{0,\1} syntax does

Reply:
i'm not sure either, I only copied everything in kevin's implementation

Reply:
I think you may have to make some changes to some random python files and run the Staged files command to see what the regex statement pulls

Reply:
You can remove the regex because python files have only 1 file type.

TypeScript files have 2 possible extensions: ".ts" and ".tsx". The regex is just there as a fancy way of capturing both of those file types.

Reply:
ah great, thanks for the clarification